RadarSDK/Include/Radar.h

```diff
@@ -489,11 +489,12 @@ typedef void (^_Nullable RadarLogConversionCompletionHandler)(RadarStatus status
  @warning Note that you must configure SSL pinning before calling this method.
 
  @param beacons A boolean indicating whether to range beacons.
+ @param desiredAccuracy The desired accuracy.
  @param completionHandler An optional completion handler.
 
  @see https://radar.com/documentation/fraud
  */
-+ (void)trackVerifiedWithBeacons:(BOOL)beacons completionHandler:(RadarTrackVerifiedCompletionHandler _Nullable)completionHandler NS_SWIFT_NAME(trackVerified(beacons:completionHandler:));
++ (void)trackVerifiedWithBeacons:(BOOL)beacons desiredAccuracy:(RadarTrackingOptionsDesiredAccuracy)desiredAccuracy completionHandler:(RadarTrackVerifiedCompletionHandler _Nullable)completionHandler NS_SWIFT_NAME(trackVerified(beacons:desiredAccuracy:completionHandler:));
```

Comment:
is there any concerns of causing confusion of `desiredAccuracy` relating to location accuracy?